bundles/org.openhab.binding.miio/src/main/resources/ESH-INF/thing/commonChannels.xml

```diff
@@ -157,6 +157,21 @@
 	<channel-type id="humidity">
 		<item-type>Number</item-type>
 		<label>Humidity</label>
+		<state pattern="%.1f" readOnly="true" />
+	</channel-type>
+	<channel-type id="pm25">
+		<item-type>Number</item-type>
+		<label>PM2.5</label>
+		<state pattern="%.1f" readOnly="true" />
+	</channel-type>
+	<channel-type id="co2">
+		<item-type>Number</item-type>
+		<label>CO2</label>
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="tvoc">
+		<item-type>Number</item-type>
+		<label>tVOC</label>
```

Comment:
Can you provide a description or a better label? I can't imagine what this channel is doing.

Reply:
Added a description. Label is fine, since that's what an actual device shows: https://i.imgur.com/O9RiEuw.jpg

Reply:
It would be good to point the unit of the value. The device send the value in response in ppb. But it can be displayed in ppm (like in your image, 1 ppm=1000 ppb) and mg/m3 (mg/m3 = 1/218.77 ppb). I was confused when I saw the value for the first time because it was completely different from the value displayed by the device (in 
mg/m3).

Reply:
I think this channel might be used by other devices in the future too. They might provide values in different units. So I think it's better keep it as it is and let a user to specify units in a sitemaps.

Reply:
It is a good reason. But it will be useful to add the information to README.md at least. Anyway the description of new supported device should be added to it. You may be the the first one who fill the remark column.

Reply:
Probably the channel should use UoM.